odoo/openupgrade/doc/source/modules120-130.rst

```diff
@@ -575,7 +575,7 @@ missing in the new release are marked with |del|.
 +----------------------------------------------+-------------------------------------------------+
 | |new| sale_product_matrix                    |                                                 |
 +----------------------------------------------+-------------------------------------------------+
-|sale_purchase                                 |                                                 |
+|sale_purchase                                 | Nothing to do                                                  |
```

Comment:
Remove extra space and I think you need to squash commits

Reply:
OK